test/unit/tracks/text-track.test.js

```diff
@@ -7,14 +7,23 @@ import TextTrack from '../../../src/js/tracks/text-track.js';
 import TestHelpers from '../test-helpers.js';
 import sinon from 'sinon';
 import log from '../../../src/js/utils/log.js';
+import XHR from 'xhr';
 
 QUnit.module('Text Track', {
   beforeEach() {
     this.tech = new TechFaker();
+    this.oldXMLHttpRequest = XHR.XMLHttpRequest;
+    this.oldXDomainRequest = XHR.XDomainRequest;
+    this.xhr = sinon.useFakeXMLHttpRequest();
+    XHR.XMLHttpRequest = this.xhr;
```

Comment:
is this not too late to override XHR? That's why previously, we were doing it in the global-shims.js file.

Reply:
these tests are broken because the rebase didn't notice a conflict between '@videojs/xhr' and 'xhr' in the import at the top. I think it should be fixed now.